content/references/http-websocket-apis/public-api-methods/server-info-methods/nft_info.md

```diff
@@ -0,0 +1,134 @@
+{
+    "method": "nft_info",
+    "params": [
+      {
+          "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C"
+      }
+    ]
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+[Try it! >](websocket-api-tool.html#nft_info)
+
+The request contains the following parameters:
+
+| `Field`        | Type                       | Description                    |
+|:---------------|:---------------------------|:-------------------------------|
+| `nft_id`       | String                     | A unique identifier for the non-fungible token (NFT). |
+| `ledger_hash`  | String                     | _(Optional)_ A 20-byte hex string for the ledger version to use. If you do not specify this field, it defaults to `validated`. (See [Specifying Ledgers][]) |
+| `ledger_index` | String or Unsigned Integer | _(Optional)_ The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. If you do not specify this field, it defaults to `validated`. Do not specify the `ledger_index` as `closed` or `current`. Specifying `current` forwards the request to the P2P `rippled` server and the `nft_info` API is not available on `rippled`. (See [Specifying Ledgers][]) | 
+
+## Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+  "result": {
+    "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C",
+    "ledger_index": 2436210,
+    "owner": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "is_burned": false,
+    "flags": 0,
+    "transfer_fee": 0,
+    "issuer": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "nft_taxon": 0,
+    "nft_sequence": 12,
+    "uri": "ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf4dfuylqabf3oclgtqy55fbzdi",
+    "validated": true,
+    "status": "success",
+    "warnings": [
+      "This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request",
+      "This server may be out of date"
+    ]
+  }
+```
+
+*JSON-RPC*
+
+```json
+{
+  "result": {
+    "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C",
+    "ledger_index": 2436210,
+    "owner": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "is_burned": false,
+    "flags": 0,
+    "transfer_fee": 0,
+    "issuer": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "nft_taxon": 0,
+    "nft_sequence": 12,
+    "uri": "ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf4dfuylqabf3oclgtqy55fbzdi",
+    "validated": true,
+    "status": "success",
+    "warnings": [
+      "This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request",
+      "This server may be out of date"
+    ]
+  }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format][], with a successful result containing an `info` object as its only field.
+
+The `info` object may have some arrangement of the following fields:
```

Comment:
`info` object? there's no `info` key. do you just mean that the response is itself an "nft_info object" by virtue of the fact that it is an object and is the response to `nft_info`?

Reply:
Perhaps refer to the `info` object as the `nft_info` response.

Reply:
Yes, will update the "info" object to "nft_info" response.